docs/architecture/decisions/0003-logic-for-signer-data-model.md

```diff
@@ -0,0 +1,70 @@
+# 1. ADR title Logic for Signer Data Model
+
+Authors: Danilo Spinelli & Filippo Tenaglia
```

Comment:
```suggestion
```

thre's no need to specify authors since its' recorded by git (moreover, it's not in the ADR template)